### PR TITLE
data: mherzog4

### PR DESCRIPTION
**Pull request by mherzog4**
## Changes proposed

Adding json data for my profile

## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3231"></a>

### CODE REVIEW

File: data/mherzog4.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://twitter.com/https://www.linkedin.com/in/mattherzog3/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Visit my website",
+      "url": "http://mattherzog.me/",
+      "icon": "fa-solid fa-code-compare"
```
Comment (eddiejaoude) [inline data/mherzog4.json:21]: These are not valid icons, please select a valid icon https://linkfree.eddiehub.io/icons

**Review by eddiejaoude — DISMISSED** [review]
Looks good, but invalid icons used for your website

File: data/mherzog4.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "avatar": "https://github.com/mherzog4.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mattherzog5",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://twitter.com/https://www.linkedin.com/in/mattherzog3/",
```
Comment (kumarsonsoff3) [inline data/mherzog4.json:15]: Please fix this invalid URL! 🛠️

```suggestion
      "url": "https://www.linkedin.com/in/mattherzog3/",
```

Comment (mherzog4) [reply data/mherzog4.json]: Sorry about that - was doing it from my phone.

Thanks for flagging.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your Contribution! 🎉 

<p align="center"></p>

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good!

**Pull request merged**